lib/ansible/modules/cloud/amazon/cloudformation_info.py

```diff
@@ -60,38 +62,48 @@
 # Note: These examples do not set authentication details, see the AWS Guide for details.
 
 # Get summary information about a stack
+- cloudformation_info:
+    stack_name: my-cloudformation-stack
+  register: output
+
+- debug:
+    msg: "{{ output['cloudformation']['my-cloudformation-stack'] }}"
+
+# When the module is called as cloudformation_facts, facts are published in
+# ansible_facts['cloudformation'][<stack_name>] and can be used as follows.
+# Note that this is deprecated will stop working in Ansible 2.13.
```

Comment:
```suggestion
# Note that this is deprecated and will stop working in Ansible 2.13.
```

Reply:
Good spot! Fixed it in 8527712f8b7588c5bcc9a9de905d34b122e3851e